portfolio/src/main/webapp/index.html

```diff
@@ -80,8 +83,21 @@ <h2 class="page_headers">Leave a comment!</h2>
             </template>
             <div id="comment-selector-container">
               <form action="/load-comments" method="GET">
-                <label for="number-of-comments"> Number of Comments: </label>
-                <input type="number" name="number-of-comments" id="number-of-comments" max="7" min="0" value="0">
+                <div>
+                  <label for="number-of-comments"> # of Comments: </label>
+                  <input type="number" name="number-of-comments" id="number-of-comments" max="7" min="0" value="0">
+                </div>
+                <br/>
+                <div>
+                  <label for="language_selector">Language: </label>
+                  <select name="language_selector" id="language_selector">
+                    <option value="en">English</option>
+                    <option value="zh">Chinese</option>
+                    <option value="es">Spanish</option>
```

Comment:
Not even really a nit, but while we're learning about languages, many systems separate languages by regions as well. For example, we differentiate between "en_us" and "en_uk" for american vs british english, (e.g. color in US vs. colour in UK). Same with many languages, including Spanish. Food for thought.

Reply:
Sure, those languages where just set as an example to test the functionality of the translation API. Currently in one of my latest commits I changed this to load the whole list of languages supported by the translation API for full diversity.